### PR TITLE
In the next milestone of my portfolio project I and my partner make the Web page accessible

### PR DESCRIPTION
**Pull request by Atifzada**
**Page titles:**
-Check the title of the portfolio
-no issues found

**Image alt:**
-Add alt to all the images in the portfolio web
-Check the alternative text to provide adequate information for the missing images

**Text headings:**
-Check the page has a heading
-Check the heading hierarchy is meaningful.
-Check all the headings accordingly and make the required changes.

**Color contrast:** 
-Check that the portfolio web colors are all according to the Figma template
-no issues found

**Resize:** 
-Check that the font sizes are all according to the Figma template
-no issues found

**Interaction:** 
-Add labels  
-set as invisible

**Moving content:** 
-Check no moving Content is found
-no issues found

**Multimedia:**
-no issues found

**The basic structure of the page:**
-check all the headings, they are right above the information they apply to.
-Check the alternative text to provide adequate information for the images
-Check that blocks of information have clear headings

# links corresponding to the line of code where changes made are here:
https://github.com/Atifzada/Portfolio-Project/pull/14#discussion_r1120257333
https://github.com/Atifzada/Portfolio-Project/pull/14#discussion_r1120258309
https://github.com/Atifzada/Portfolio-Project/pull/14#discussion_r1120259453
https://github.com/Atifzada/Portfolio-Project/pull/14#discussion_r1120260380
https://github.com/Atifzada/Portfolio-Project/pull/14#discussion_r1120262370
https://github.com/Atifzada/Portfolio-Project/pull/14#discussion_r1120263108
https://github.com/Atifzada/Portfolio-Project/pull/14#discussion_r1120264725

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -11,7 +11,7 @@
     </style>
     <link rel="stylesheet" href="style.css" />
     <link rel="stylesheet" href="index.css" />
-    <title>Protfolio Web</title>
+    <title>Portfolio-Web</title>
```
Comment (Atifzada) [inline index.html:14]: Change the web title

File: index.html [hunk 2]
```diff
@@ -23,7 +23,7 @@
                 <li><a href="#">About</a></li>
                 <li><a href="#">Contact</a></li>
             </ul>
-            <img class="hamburg" src="assets/hamburg.svg" alt="hamburg" />
+            <img class="hamburg" src="assets/hamburg.svg" alt="hamburger image" />
```
Comment (Atifzada) [inline index.html:26]: Change the alt to descriptive one

File: index.html [hunk 3]
```diff
@@ -72,7 +72,7 @@ <h1 class="title">I’m Atif<br>
                 <img class="pic" src="./assets/works img/work1.svg" alt="card1" />
             </div>
             <div class="works-details">
-                <h1 class="comp-title">Tonic</h1>
+                <h2 class="comp-title">Tonic</h2>
```
Comment (Atifzada) [inline index.html:75]: Change the heading according to the web accessibility requirments

File: index.html [hunk 4]
```diff
@@ -96,11 +96,11 @@ <h1 class="comp-title">Tonic</h1>
         </div>
         <div class="card reverse">
             <div class="portfolio-img pic2">
-                <img class="pic" src="./assets/works img/work4.svg" alt="card1" />
+                <img class="pic" src="./assets/works img/work4.svg" alt="card2" />
```
Comment (Atifzada) [inline index.html:99]: Change the alt the suitable alt

File: index.html [hunk 5]
```diff
@@ -96,11 +96,11 @@ <h1 class="comp-title">Tonic</h1>
...
             </div>
             <div class="works-details">
-                <h1 class="comp-title">Multi-Post
-                    Stories</h1>
+                <h2 class="comp-title">Multi-Post
+                    Stories</h2>
```
Comment (Atifzada) [inline index.html:103]: Change the heading to follow the correct hierarchy

File: index.html [hunk 6]
```diff
@@ -123,11 +123,11 @@ <h1 class="comp-title">Multi-Post
         </div>
         <div class="card">
             <div class="portfolio-img pic3">
-                <img class="pic" src="./assets/works img/work2.svg" alt="card1" />
+                <img class="pic" src="./assets/works img/work2.svg" alt="card3" />
```
Comment (Atifzada) [inline index.html:126]: Change the alt the suitable alt

File: index.html [hunk 7]
```diff
@@ -150,10 +150,10 @@ <h1 class="comp-title">Facebook 360
         </div>
         <div class="card reverse">
             <div class="portfolio-img pic4">
-                <img class="pic" src="./assets/works img/work3.svg" alt="card1" />
+                <img class="pic" src="./assets/works img/work3.svg" alt="card4" />
             </div>
             <div class="works-details">
-                <h1 class="comp-title">Uber Navigation</h1>
+                <h2 class="comp-title">Uber Navigation</h2>
```
Comment (Atifzada) [inline index.html:156]: Change the heading to follow the correct hierarchy

**Review by mckent05 — CHANGES_REQUESTED** [review]
Hi @Atifzada, 

# STATUS: CHANGES REQUIRED ♻️ ♻️ 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
almost there

## Highlights
- All linter checks are passing ✔️ ✔️ 
- Your PR is professional 👍 👍 


## Required Changes ♻️
- According to the project requirements, `If issues were found, they should link to the corresponding line of code in the Pull Request changes`. Please include the links in your PR summary. Please fix this 😄 
_Check the comments under the review._

## Optional suggestions
Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by Atifzada** [comment]
All the links to the corresponding line where changes are made are added in PR summary as per the requirement

**Pull request merged**